modules/ballerina-core/src/main/java/org/ballerinalang/services/dispatchers/websocket/WebSocketResourceDispatcher.java

```diff
@@ -69,7 +70,7 @@ public Resource findResource(Service service, CarbonMessage cMsg, CarbonCallback
             throw new BallerinaException("Error occurred in WebSocket resource dispatchers : " + e.getMessage(),
                                          balContext);
         }
-        throw new BallerinaException("No matching Resource found for dispatchers.");
+        throw new ResourceNotFoundException("No matching Resource found for dispatchers.");
```

Comment:
As I see in ServerConnectorMessageHandler.java:71 "service" has been written in lowercase. Shouldn't we follow the same here for "resource"?